### DISCUSSION TITLE
Embedded MathML

### DISCUSSION THREAD

**Discussion by franklinvp** [typst/typst#208] [Q&A]
Does `typst` allow to have embedded MathML?

I have a use case in which it would be useful to have a markup language that converts to PDF, that can support (a large volume of legacy) formulas that are in MathML together with (new) formulas in LaTeX.

**Comment by jonsterling** [comment]
I would also find this useful. It's worth nothing that ConTeXT supports this, and it is really powerful... MathML is not nice to write, but it is easy for tools to emit — and it is very helpful if the output of such tools can be used in a variety of typesetting systems.

**Comment by SeSodesa** [comment]
Initial work is being performed here: <https://github.com/typst/typst/pull/7436>.
